samples/framework/libraries/migrate-library-csproj-keep-existing/src/Car/Car.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <AssemblyName>Car</AssemblyName>
+    <PackageId>Car</PackageId>
+    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
```

Comment:
I had migrated these from xproj to csproj using the [dotnet-migrate tool](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-migrate).  You're right, they are not necessary. I'll remove these.